modules/mobile/app/workers/mobile/v0/vet360_linking_job.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+
+module Mobile
+  module V0
+    class Vet360LinkingJob
+      include Sidekiq::Worker
+
+      sidekiq_options(retry: false)
+
+      def perform(current_user)
+        Mobile::V0::Profile::SyncUpdateService.new(current_user).await_vet360_account_link
```

Comment:
Don't forget to release the lock (on success or fail, we're only locking the attempt)